core/modules/user/user.install

```diff
@@ -1344,6 +1352,45 @@ function user_update_1023() {
   }
 }
 
+/**
+ * Add changed field to users table.
+ */
+function user_update_1024() {
+  // The "changed" column was renamed to "access" in system_update_136(), and
+  // the "changed" index on "access" column may persist on old MySQL databases.
```

Comment:
@argiepiano offered an explanation in https://github.com/backdrop/backdrop-issues/issues/5944#issuecomment-1412149578, so how about we change this inline to something like this?:

```suggestion
 * Add a "changed" column to the "users" table.
 */
function user_update_1024() {
  // On older databases originating from Drupal 5, the "changed" column was
  // renamed to "access" in system_update_136(), and the "changed" index on the
  // "access" column may still persist. That column index needs to be removed
  // before adding the new "changed" column.
  // See: https://api.drupal.org/api/drupal/modules%21system%21system.install/function/system_update_136/5.x
```